hack/mutation/conversion_webhooks_injection.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Add the conversion stanza to the CRD spec to enable conversion via webhook
+yq eval '.spec.conversion = {"strategy": "Webhook", "webhook": {"conversionReviewVersions": ["v1beta1", "v1"], "clientConfig": {"service": {"name": "karpenter", "namespace": "kube-system", "port": 8443}}}}' -i pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml
```

Comment:
This strikes me as something that I'm surprised that we would need. Does kubebuilder not have a way to specify this with markers?

Reply:
+1 on this, but also kind of makes sense? kubebuilder could do it by appending the conversionReviewVersions references across annotations, but also not sure if it's a feature that's just not possible

Reply:
I don't think there's any way for it to know of the port or service name at generation time. I think in an ideal world we would template our CRDs as part of our helm chart, but this probably isn't worth the squeeze since these will only be included for a single minor version.